Facebook.Android/source/facebook-core/facebook-core.csproj

```diff
@@ -56,7 +56,7 @@
   <ItemGroup>
     <PackageReference Include="Xamarin.AndroidX.Annotation" Version="1.1.0.1" />
     <PackageReference Include="Xamarin.AndroidX.Legacy.Support.Core.Utils" Version="1.0.0.1" />
-    <PackageReference Include="Adjust.Xamarin.Android.InstallReferrer" Version="1.0.1" />
+    <PackageReference Include="Xamarin.Android.Binding.InstallReferrer" Version="2.2.0" />
```

Comment:
@Redth This was done in that way, because Adjust is unable to reference 3rd party nugets by their own policy. So i just told them to publish my bindings for the installreferrer as nuget themselfes, to have it usable here (as Xamarin.Android.Binding.InstallReferrer does not seem to be the official one either).

Unfortunately, their (Adjusts) policy blocks progress, and they just do not start reviewing my PR (even though promised otherwise).

Reply:
As Adjust refuses to adapt any of the suggested changes (https://github.com/adjust/xamarin_sdk/pull/33), I am perfectly fine with this change. In addition, I removed the obsolete NuGet.Config file.